src/utils/validators/graphqlQueryValidators.ts

```diff
@@ -99,9 +99,17 @@ export const createDonationQueryValidator = Joi.object({
     .required()
     .valid(...Object.values(NETWORK_IDS)),
   tokenAddress: Joi.when('transactionNetworkId', {
-    is: 0, // if its solana network
+    is: NETWORK_IDS.SOLANA_MAINNET,
```

Comment:
@CarlosQ96  IT doesnt need to add this exception for SOLANA DEVNET and TESTNET as well?

Reply:
Its in the otherwise clauses.

Reply:
Oh you are right

Reply:
First, I think there would be a better solution than nesting, like 
```suggestion
    is: Joi.valid(NETWORK_IDS.SOLANA_MAINNET, NETWORK_IDS.SOLANA_DEVNET, NETWORK_IDS.SOLANA_TESTNET),
```

Beyond that, I guess it's better to decide the valid regext to match based on chainType rather than networkId

Reply:
Done